_posts/2020-03-23-Steering-Group-executive-summary,-action-items,-and-minutes.md

```diff
@@ -0,0 +1,74 @@
+---
+title: Steering Group executive summary, action items, and minutes
+author: Franklin Feingold
+display: true
+---
+
+# Steering Group executive summary, action items, and minutes
+
+Date: Thursday March 5, 2020
+Attended: Franklin Feingold, Melanie Ganz, Guiomar Niso, Robert Oostenveld
```

Comment:
insert a blank line in between line 9 and 10, please, otherwise this gets rendered as a single line